javascript/Dockerfile

```diff
@@ -1,5 +1,11 @@
-FROM node:12-slim
+FROM node:lts-slim
```

Comment:
Switched to LTS in hopes of reducing the maintenance burden. Could lead to silent breakage further down the line if we don't keep dependencies up-to-date though.